sporkfish/evaluator.py

```diff
@@ -1,10 +1,8 @@
 from enum import Enum
-from typing import Callable
 
 import chess
 
 from .board.board import Board
-from .configurable import Configurable
```

Comment:
Again, want to check - did you do these manually or was it done automatically by ruff/pre-commit magic?

Reply:
magic